storage/ceph.py

```diff
@@ -1,14 +1,22 @@
 import pyarrow.parquet as pq
-import s3fs
+import s3fs, datetime, os, s3, local
```

Comment:
I dont think we'll need `s3` and `local` imported.

Reply:
ok removing those.